TourMate/TourMate/Frontend/Plan/Views/AddPlanView/Accommodation/AddAccommodationView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AddAccommodationView.swift
+//  TourMate
+//
+//  Created by Tan Rui Quan on 9/4/22.
+//
+
+import SwiftUI
+
+struct AddAccommodationView: View {
+    @StateObject var viewModel: AddAccommodationViewModel
+    var dismissAddPlanView: DismissAction
+
+    var body: some View {
+        AddPlanView(viewModel: viewModel,
+                    dismissAddPlanView: dismissAddPlanView,
+                    planType: "Accommodation") {
+            PlanFormView(viewModel: viewModel,
+                         startDateHeader: "Check-in Date",
+                         endDateHeader: "Check-out Date") {
+
+                Section("Location") {
```

Comment:
Can we further extract PlanFormView(){ } into the individual Plan's form views? So that the XXPlanFormView can be reused for add and edit?

Reply:
Thanks for the suggestion, initially i thought i wasnt possible cuz the add and edit view models are separate

Reply:
Now its much cleaner